### DISCUSSION TITLE
(preset) nlo's vhs preset for ntsc-rs

### DISCUSSION THREAD

**Discussion by dawnch0rus** [ntsc-rs/ntsc-rs#77] [Presets]
![philipscard_ntsc_ntsc2](https://github.com/valadaptive/ntsc-rs/assets/154479340/f5cf2856-a65c-448d-8c28-e9c309ffcca5)

been working on this a bit and i'm fairly happy with how it looks. it's late so i dont feel like getting any other screenshots but hopefully the test card should show everything you'd be interested in. this is my personal vhs preset, modeled after the output of my vcr.

by default, sharpening is disabled. if you want to simulate vhs hq sharpening, just tick the "sharpen" box - the default settings should look good but feel free to play around with them.

[ntsc-rs_vhs-sp_v8.json](https://github.com/valadaptive/ntsc-rs/files/15058389/ntsc-rs_vhs-sp_v8.json)

**Comment by egordobrij3-jpg** [comment]
34

**Comment by egordobrij3-jpg** [comment]
134677

**Comment by egordobrij3-jpg** [comment]
https://github.com/user-attachments/assets/fe0fc4b7-1a87-4675-932c-ca8db44b4d77

3
